### PR TITLE
Feature/loader

### PR DESCRIPTION
**Pull request by DanSallau**
loader

### CODE REVIEW

File: lib/src/loader/loader.component.scss [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    }
+}
+
+$offset: 200;
+$duration: 1.5s;
+$duration-sq: 1.2s;
+$loader-xs: 1rem;
+$loader-sm: 2.125rem;
+$loader-md: 4.063rem;
+$loader-lg: 6.25rem;
```
Comment (mario-subo) [inline lib/src/loader/loader.component.scss:17]: image
Please use the existing SEB Bootstrap variables where possible

File: lib/src/loader/loader.component.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<div class="{{ className }}" [ngClass]="size | loaderClasses: cover:fullscreen:type:backdrop" [attr.role]="role">
```
Comment (mario-subo) [inline lib/src/loader/loader.component.html:1]: wouldn't this result in a class named undefined if you don't pass a custom className?

File: lib/src/loader/loader.component.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    /** Dims the background the background to indicate UI interactions are blocked */
+    @Input() backdrop?: boolean;
+    /** Screen reader text. Default is `Loading...` */
+    @Input() srText?: string;
+    /** Show or hide the loader. Default is `true` */
+    @Input() toggle?: boolean;
+
+    @Input() role?: string = "status";
+
+    @Input() className?: string;
```
Comment (yousifalraheem) [inline lib/src/loader/loader.component.ts:30]:
```suggestion
    /** HTML role value. Default is `status`  */
    @Input() role?: string = "status";
    /** Element class name */
    @Input() className?: string;
```

**Pull request merged**